### DISCUSSION TITLE
Using Tailwind with PostCSS and Webpack (Latest)

### DISCUSSION THREAD

**Discussion by ThePlayerSD** [tailwindlabs/tailwindcss#2427] [Help] [answered]
Hey there,

So I'm trying to set up a fresh Webpack with TailwindCSS.
I've got it running PostCSS find from Webpack and including the correct Style Files.
My issue is when I try to build, PostCSS compiles fine, but when I include Tailwind I get an error:

`Unknown error from PostCSS plugin. Your current PostCSS version is 7.0.34, but postcss-functions uses 6.0.23. Perhaps this is the source of the error below.`

I thought it was a config issue, but even using the init config or no config at all causes the same issue, and I'm super confused.

Let me know if there is anything more you need and I look forward to hearing from you!

Thanks!

[assets.zip](https://github.com/tailwindlabs/tailwindcss/files/5261581/assets.zip)

**Comment by JNavith** [comment, marked as answer]
I had the same problem. You **might** solve it by downgrading PostCSS to `8.0.6`.

**Reply by ThePlayerSD** [reply]
Thank you! It's not erroring. Does that mean that this an issue of decrepitation of the functions that Tailwind uses?

**Comment by ThePlayerSD** [comment]
@adamwathan Am I right in saying that this is due to the new version of PostCSS not being supported yet?

**Reply by adamwathan** [reply]
Try upgrading to PostCSS 8.0.8

**Reply by ThePlayerSD** [reply]
PostCSS 8.0.8 causes the issue above :)  (further debug below)

```ERROR in ./src/css/style.css (./node_modules/css-loader/dist/cjs.js??ref--4-1!./node_modules/postcss-loader/dist/cjs.js??ref--4-2!./src/css/style.css)
    Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
    TypeError: Cannot read property 'type' of undefined

**Reply by ThePlayerSD** [reply]
Hey @adamwathan,

I'm fairly sure this is still an issue.
It's fine with PostCSS 8.0.6 but fails on PostCSS 8.0.8.
Any more info you need from me to help resolve this?

It could be my bad I'm not sure if I've done something wrong

**Comment by adamwathan** [comment]
Can you please provide a GitHub repo that reproduces so I can test your setup?
